src/utils/analysis-utils.ts

```diff
@@ -434,8 +435,31 @@ export async function loadFeaturesClientSide(
 }
 
 export function getAnalysisTableColumns(
-  analysisResult: BaselineLayerResult,
+  analysisResult: BaselineLayerResult | PolygonAnalysisResult,
 ): Column[] {
+  if ('tableColumns' in analysisResult) {
+    return analysisResult.tableColumns.map(column => {
+      return {
+        id: column as any,
+        // remove prefix from column labels
+        // example: replace "stat:area" with "area"
+        label: column.replace(/^[a-z]+:/i, ''),
+        format: (value: number | string) => {
+          if (typeof value === 'number') {
+            return value.toLocaleString('en-US');
+          }
+          if (value === null) {
+            return 'null';
+          }
+          if (value === undefined) {
+            return 'null';
+          }
+          return value;
```

Comment:
this could simply be

`return isNil(value) ? 'null' : value`

Reply:
awesome. will do